#line_point3d
obst_avoid/point3d[] line_p3d
